app/controllers/volunteer_ops_controller.rb

```diff
@@ -57,15 +69,15 @@ def volunteer_op_params
 
   private
 
-  def build_map_markers(organisations)
-    ::MapMarkerJson.build(organisations) do |org, marker|
+  def build_map_markers(organisations,  type = :harrow, include_extra_organisation_data = true)
+    ::MapMarkerJson.build(organisations, include_extra_organisation_data) do |org, marker|
       marker.lat org.latitude
       marker.lng org.longitude
-      marker.infowindow render_to_string( partial: 'popup', locals: {org: org})
+      marker.infowindow render_to_string( partial: "popup_#{type}", locals: {org: org})
       marker.json(
         custom_marker: render_to_string(
           partial: 'shared/custom_marker',
-          locals: { attrs: [ActionController::Base.helpers.asset_path("volunteer_icon.png"),
+          locals: { attrs: [ActionController::Base.helpers.asset_path("volunteer_icon_#{type}.png"),
```

Comment:
Metrics/LineLength: Line is too long. [100/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)